src/content/collections.ts

```diff
@@ -55,7 +55,12 @@ const collections = {
           alt: z.string(),
         })
         .optional(),
-      cfp: z.object({ href: z.string().url() }).optional(),
+      cfp: z
+        .object({
+          href: z.string().url(),
+          endDate: z.string(),
```

Comment:
I think we won't always have the end date.

```suggestion
          endDate: z.string().optional(),
```

Reply:
Also, maybe it can be parsed as a date, so we do not need to make `new Date(props.event.cfp.date)` and just use `props.event.cfp?.date`


```suggestion
          endDate: z.date().optional(),
```